### ISSUE TITLE
Normal MODIFIED mode

### ISSUE THREAD

**Issue by RedHatDarrow** [sethen/line-number-change-mode.nvim#1]
Would really helpful if you add a color to indicate a modified buffer in normal mode.

**Comment by sethen** [comment]
Currently this just detects mode changes... What would be the desired behavior for a modified buffer in the line number?

**Comment by RedHatDarrow** [comment]
> Currently this just detects mode changes... What would be the desired behavior for a modified buffer in the line number?

The desired behavior for indicating a modified buffer in the line number is simply a visual cue that the current buffer has been changed. This aims to prevent accidental modifications in the code, especially when you might alter something without noticing and then unintentionally save everything together, including unwanted changes.

I usually have an auto-command for this, but it would be great if the plugin could integrate this functionality comprehensively. I use red to indicate a modified normal mode and green for an unmodified normal mode, while other modes remain unaffected. It would also be beneficial to include a toggle for users who might not want or like this feature.

**Comment by sethen** [comment]
@RedHatDarrow So this is just for modified buffers and not for something that's version controlled?  There are numerous plugins available for seeing what's been modified via git in Neovim.  I am assuming you want something right on your line number though... right?

**Comment by RedHatDarrow** [comment]
> @RedHatDarrow So this is just for modified buffers and not for something that's version controlled?  There are numerous plugins available for seeing what's been modified via git in Neovim.  I am assuming you want something right on your line number and just for modified buffers that aren't version controlled though... right?

Yes, that's exactly it. Just a visual marker indicating that the current buffer has been modified, both in files tracked by Git and in files that aren't part of any version control system.
